tests/integration/conftest.py

```diff
@@ -90,7 +90,7 @@ def app(tmpdir, request):
 
 
 @pytest.fixture(scope='function')
-def client_toggled(app, request):
+def client_toggled(config, client, request):
```

Comment:
That's really neat. I didn't know pytest-flask had a config fixture.

It looks like the app fixture loads some test config. Do you know where the config fixture is getting it's information from, and can we be sure these configs will be the same between tests?

Reply:
`app.config` (just shorthand, dependent on app fixture, and functionally scoped):
https://github.com/pytest-dev/pytest-flask/blob/master/pytest_flask/fixtures.py#L168
:arrow_up: full of goodies. 

Also, this gets us closer to being able to parameterize on different configurations, a good use case for gdc<-> planx testing not breaking each other (eventually). Instead of this wonky toggling.

Eventually, I would like to work to a model in where we are deploying the F5 buckets as different applications within an application factory so each of their configs could be separated into their own "service" behind a uwisgi+gunicorn emperor service. Could help with the apache thread explosion we currently have going on too.